bdd-generator/src/main/resources/db/changelog/db.changelog-entities.xml

```diff
@@ -214,4 +214,32 @@
             </column>
         </createTable>
     </changeSet>
+
+    <changeSet author="" id="create_default_roles">
+        <insert tableName="ROLE">
+            <column name="NAME" value="ADMIN"/>
+        </insert>
+        <insert tableName="ROLE">
+            <column name="NAME" value="TEST_ENGINEER"/>
+        </insert>
+        <insert tableName="ROLE">
+            <column name="NAME" value="TEST_LEAD"/>
+        </insert>
+        <insert tableName="ROLE">
+            <column name="NAME" value="GUEST"/>
+        </insert>
+    </changeSet>
+
+    <changeSet author="" id="create_default_admin">
+        <insert tableName="USER">
+            <column name="EMAIL" value="admin@mail.com"/>
+            <column name="PASSWORD" value="$2a$10$b8myM0VkLZTkSAgQ2Ijs.e9t.WGvXVnPKtJDQMZ7iJWdZ6uVZRX6u"/>
```

Comment:
Please document default admin login and password in project files, i.e. README.md

Reply:
done